terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
```

Comment:
Those are the data source docs--they're for importing information from external resources or another terraform confirmation: https://www.terraform.io/docs/configuration/data-sources.html. We want the resource docs, which actually provision infrastructure.